shortcuts.sh

```diff
@@ -132,19 +129,19 @@ function swgjs_create_amp_release_pr() {
 
   __swgjs_get_github_username
 
-  SWG_VERSION=$1
-  if [[ -z "$SWG_VERSION" ]]; then
-    echo "What SwG Version are you releasing? (ex: 0.1.22.333)"
-    read SWG_VERSION
-  fi
-
   swgjs_install
   __swgjs_install_amp
 
-  # Build Swgjs for AMP.
-  cd $SWGJS_PATH
+  # Switch to clean AMP branch.
   git fetch team
-  git checkout team/main
+  git checkout team/amp
+  if [[ `git status --porcelain` ]]; then
+    # Avoid building with local changes.
+    return
```

Comment:
return value should be non-zero to indicate failure.

Reply:
Good call! I just created https://github.com/subscriptions-project/swg-js/pull/2488 to fix this